BUILD

```diff
@@ -2272,4 +2272,28 @@ grpc_cc_library(
     ],
 )
 
+grpc_cc_library(
```

Comment:
We'll need to mirror this into the build.yaml, but we can cross this bridge when you're at this point of the development. Maybe add a TODO to remind yourself of it?

Reply:
Done